packages/patternfly-4/react-core/src/components/Dropdown/DropdownMenu.tsx

```diff
@@ -31,11 +31,11 @@ export interface DropdownMenuItem extends React.HTMLAttributes<any> {
   isDisabled: boolean;
   disabled: boolean;
   isHovered: boolean;
-  ref: React.RefObject<any>;
```

Comment:
Changing this type here wont break anyone?

Reply:
The refs are used internally so I don't think it should